lib/docker.js

```diff
@@ -141,21 +153,21 @@ function createDockerFile() {
     }
 
     if (opts.uid !== 0) {
-        contents += 'RUN groupadd -g ' + opts.gid + ' -r rungroup && ' +
-        'useradd -m -r -g rungroup -u ' + opts.uid + ' runuser\n' +
+        contents += 'RUN groupadd -r rungroup && ' +
```

Comment:
This should not go into this PR. Moreover, the UID/GID must match the local user's, which is especially important for the deploy procedure. Please remove the changes of this and the line below it.